Automattic/wp-calypso
# AI Logo generator: Add history carousel linked to the store

Closes #85785

## Proposed Changes

* Adds the HistoryCarousel component
* Links it to the store
* Links the Generate button to the store

## Testing Instructions



* Go to the AI logo generator modal
* Add a couple sample logos by clicking on "Generate"
* Check that the carousel is displayed and it works
* Click on a logo
* Check that the logo displayed on the main presenter component is updated accordingly

## Pre-merge Checklist
